src/ms-review.tex

```diff
@@ -190,57 +190,31 @@
-	sky. It offers a variety of background estimation methods for spectral, spatial
-	and spectro-morphological analysis. Counts, background and IRFs data are
-	bundled in datasets and can be serialised, rebinned and stacked. \gammapy
-	supports to model binned data using Poisson
-	maximum likelihood fitting. It comes with built-in spectral, spatial and
-	temporal models, support for particle radiation emission spectra as well as
-	support for custom user models, to model e.g.,
-	energy dependent morphology of \gammaray sources. Multiple datasets can be
-	combined in a joint-likelihood approach to either handle time dependent IRFs,
-	different classes of events or combination of data from multiple instruments.
-	Gammapy also implements methods to estimate flux points, likelihood
-	profiles per energy bin, light curves as well as flux and signficance maps in
-	energy bins.  We further describe the general
-	development approach and how \gammapy integrates into ecosystem of other
-	scientific and astronomical Python packages. We also present analysis examples
-	with simulated CTA data and provide results of scientific validation analyses
-	using data of existing instruments such as \hess and \fermi.
-
-	% The \gammaray can be considered as the last frontier
-	% In the last two decades the measurement of \gammaray emission from the universe has
-	% evolved from a niche
-
-	% aims heading (mandatory)
-	%{Gammapy aims}
-	%Reproducible analyses, share algorithms etc.
-
-	% methods heading (mandatory)
-	%{Gammapy methods}
-
-	% results heading (mandatory)
-	%{Gammapy results}
-
-	% conclusions heading (optional), leave it empty if necessary
-	%{Gammapy conclusions}
+
+	{Context} In the last two decades the measurement of \gammaray emission from the universe has
+	evolved from a niche area of closed collaborations operating with proprietary data and software into
+	an active and exciting field of astrophysics. The next generation of \gammaray instruments,
+	such as the Cherenkov Telescope Array (CTA), will be operated as open observatories with publicly available
+	data, which will require open high-level astronomy software customised for high energy astrophysics.
+
+	{Aims} In this article, we present the first the Long-term Support (LTS) release of \gammapy v1.0. \gammapy
+	is a community-developed, open-source Python package for \gammaray astronomy. Integrated into the general
+	python eco-system, it is built on Numpy, Scipy and Astropy and uses open FITS based data formats for serialisation.
+	It provides a uniform platform for reduction and modelling of data from many \gammaray instruments.
+
+	{Methods} Starting from event list and instrument response functions at the so called "Data Level 3",
+	\gammapy provides pipelines for reduction of the input data to binned WCS, HEALPix or region based data structures.
+	A variety of background reduction methods, including traditional techniques like the Reflected and Ring regions,
```

Comment:
To facilitate the data reduction, several technique for background estimation are implemented in the package. 
After the data are binned, the flux of one or more gamma-ray sources can be estimated using different likelihood methods and assuming a variety of built-in spectral, temporal, and spatial models.

P.S. what is novel in the 3D field-of-view likelihood technique?